### ISSUE TITLE
Emscripten imposes a license to the output

### ISSUE THREAD

**Issue by juj** [emscripten-core/emscripten#11258]
After a recent compiler update we have started getting an Emscripten-imposed license text to all our output files:

```
emcc tests/small_hello_world.c -o a.html -s MINIMAL_RUNTIME=1
```
the output `a.js` will read

```js
if(typeof Module==="undefined"){var Module={};}/**
d
 * @license
 * Copyright 2010 The Emscripten Authors
 * SPDX-License-Identifier: MIT
 */

var Module = Module;


var ENVIRONMENT_IS_NODE = typeof process === 'object';

var ENVIRONMENT_IS_SHELL = typeof read === 'function';

var ENVIRONMENT_IS_WEB = !ENVIRONMENT_IS_NODE && !ENVIRONMENT_IS_SHELL;

if (ENVIRONMENT_IS_NODE && ENVIRONMENT_IS_SHELL) {
  throw 'unclear environment';
}

// Wasm or Wasm2JS loading:
if (ENVIRONMENT_IS_NODE) {
  var fs = require('fs');

...
```

Note also the stray 'd' character in the license block.

What is this about? IIUC the software license of Emscripten compiler should not dictate the license of the output itself? E.g. GCC is GPL, but one should not distribute everything compiled with GCC as GPL as well. And Emscripten certainly has a more permissive license, so I don't expect that it makes claims about the license of the output?

Running with 
```
emcc tests/small_hello_world.c -o a.html -s MINIMAL_RUNTIME=1 -s EMIT_EMSCRIPTEN_LICENSE=0
```
does not help the case, but the license block still remains.

**Comment by juj** [comment]
Fix in #11258, that does not look intentional.

**Comment by kripken** [comment]
Correct, that's not intentional, `EMIT_EMSCRIPTEN_LICENSE` is what should control that. Looks like a bug with minimal runtime interaction with it.
